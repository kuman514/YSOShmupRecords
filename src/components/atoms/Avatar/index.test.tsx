import React from 'react';
import { cleanup, render } from '@testing-library/react';
import { beforeEach, describe, expect, it } from 'vitest';
import 'jest-styled-components';

import { Avatar } from '.';

describe('TitleWithAvatar', () => {
  beforeEach(() => {
    cleanup();
  });

  it('should have a snapshot match for its appearance', () => {
    const { container } = render(
      <Avatar pxSize={150} imageUrl="Temp Image Url" />
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
