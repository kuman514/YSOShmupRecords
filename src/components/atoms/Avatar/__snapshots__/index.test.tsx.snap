// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TitleWithAvatar > should have a snapshot match for its appearance 1`] = `
.c0 {
  display: flex;
  justify-content: center;
  align-items: center;
  border-radius: calc(150px / 2);
  width: 150px;
  height: 150px;
  background-color: var(--gray-color);
}

.c1 {
  color: #000000;
  font-size: 16px;
  font-weight: 500;
  border-radius: inherit;
  width: inherit;
  height: inherit;
}

<div
  class="c0"
>
  <img
    alt="YSOShmupRecords avatar"
    class="c1"
    src="Temp Image Url"
  />
</div>
`;
